src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,498 @@
+---
+title: Monitor your Flutter app
+tags:
+    - Flutter
+    - Mobile monitoring
+    - New Relic Mobile Flutter
+    - Get started
+metaDescription: "New Relic's Flutter agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.png'
+
+Monitor your Flutter application with our New Relic Flutter agent. Capture Dart errors, track specific network requests, and use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
+
+<img
+  title="Flutter summary view in the UI"
+  alt="Summary view of a flutter app in New Relic"
+  src={mobileFlutterSummary}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile >** (select an app) **> Summary**: View Flutter data, track Http requests and errors, and monitor how your app is performing over time.
+</figcaption>
+
+Ready to get started? Follow the steps below! 
+
+
+## Step 1: Review the requirements
```

Comment:
I would reformat this entire procedure into the steps component we now have. Should just take a minute or two, it's pretty easy! https://docs.newrelic.com/docs/style-guide/structure/steps/